src/files.js

```diff
@@ -53,21 +53,27 @@ module.exports = (common) => {
       describe('.add', () => {
         it('stream', (done) => {
           const buffered = new Buffer('some data')
+          const expectedMultihash = 'QmVv4Wz46JaZJeH5PMV4LGbRiiMKEmszPYY3g6fjGnVXBS'
+
           const rs = new Readable()
           rs.push(buffered)
           rs.push(null)
 
           const arr = []
-          const filePair = {path: 'data.txt', content: rs}
+          const filePair = {
+            path: 'data.txt',
+            content: rs
+          }
+
           arr.push(filePair)
 
           ipfs.files.add(arr, (err, res) => {
             expect(err).to.not.exist
             expect(res).to.be.length(1)
-            expect(res[0].path).to.equal('data.txt')
-            expect(res[0].node.size()).to.equal(17)
-            const mh = 'QmVv4Wz46JaZJeH5PMV4LGbRiiMKEmszPYY3g6fjGnVXBS'
-            expect(bs58.encode(res[0].node.multihash()).toString()).to.equal(mh)
+            const file = res[0]
+            expect(file.path).to.equal('data.txt')
+            expect(file.size).to.equal(17)
+            expect(file.hash).to.equal(expectedMultihash)
```

Comment:
Changed the first case to the first one, then I changed the remaining ones to the second options, but those got angry at me:

```
      Uncaught AssertionError: expected { Object (path, hash, ...) } to have key 'QmVvjDy7yF7hdnqE8Hrf4MHo5ABDtb5AbX6hWbD3Y42bXP'
      + expected - actual

       [
      -  "hash"
      -  "path"
      -  "size"
      +  "QmVvjDy7yF7hdnqE8Hrf4MHo5ABDtb5AbX6hWbD3Y42bXP"
       ]

```

It seems that .to.have.key expects that the arguments are the keys to be looked inside the object and not their values. Checked the Chai API docs and didn't find an example to check for a key and then assert its value (http://chaijs.com/api/bdd/), keeping those how they were :)